eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/IntegerConverter.php

```diff
@@ -69,10 +69,20 @@ public function toStorageFieldDefinition(FieldDefinition $fieldDef, StorageField
     {
         if (isset($fieldDef->fieldTypeConstraints->validators[self::FLOAT_VALIDATOR_IDENTIFIER]['minIntegerValue'])) {
             $storageDef->dataInt1 = $fieldDef->fieldTypeConstraints->validators[self::FLOAT_VALIDATOR_IDENTIFIER]['minIntegerValue'];
+            // @deprecated false constraint value, see https://jira.ez.no/browse/EZP-26382
+            // @TODO: remove false to null conversion
+            if ($storageDef->dataInt1 === false) {
```

Comment:
Like above about deprecation. In addition, messages about deprecation should include the version it must be removed at, or else it will be a mess.